### PR TITLE
Rfe trello 2146 update implementation of dom capture to use the latest dom capture js script

### PR DESCRIPTION
**Pull request by ikseek**

### CODE REVIEW

File: eyes_selenium/applitools/selenium/capture/dom_capture.py [hunk 1]
```diff
@@ -1,256 +1,194 @@
...
+
+
+def recurse_frames(driver, missing_frames_list, css_downoader):
+    # type: (EyesWebDriver, List[Text], CssDownloader) -> Dict[Text, Text]
+    frame_data = {}
+    switch_to = driver.switch_to
+    fc = driver.frame_chain.clone()
+    for missing_frame_line in missing_frames_list:
+        logger.info("Switching to frame line: {}".format(missing_frame_line))
+        original_location = _frame_location(driver)
```
Comment (ikseek) [inline eyes_selenium/applitools/selenium/capture/dom_capture.py:111]: Rename to frame_url

File: eyes_selenium/applitools/selenium/capture/dom_capture.py [hunk 2]
```diff
@@ -1,256 +1,194 @@
...
+    frame_data = recurse_frames(driver, missing_frames, css_downoader)
+    return efficient_string_replace(
+        separators.iframe_start_token, separators.iframe_end_token, data[0], frame_data
+    )
+
+
+def get_dom(driver):
+    # type: (EyesWebDriver) -> Text
+    with CssDownloader() as css_downloader:
+        original_fc = driver.frame_chain.clone()
```
Comment (ikseek) [inline eyes_selenium/applitools/selenium/capture/dom_capture.py:159]: Maybe make context manager from it

File: eyes_selenium/package.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   "name": "node_resources",
   "description": "required javascript scripts",
   "dependencies": {
-    "@applitools/dom-capture": "=7.2.6",
+    "@applitools/dom-capture": "=7.2.4",
```
Comment (ikseek) [inline eyes_selenium/package.json:5]: Set back to 7.2.6

File: tests/functional/eyes_selenium/selenium/test_dom_capture.py [hunk 4]
```diff
@@ -155,6 +155,18 @@ def test_position_scrolled_to_origin_after_traversing(self):
         )
         assert current_scroll == Point(0, 0)
 
+    @pytest.mark.test_page_url("https://corsat2.herokuapp.com")
```
Comment (ikseek) [inline tests/functional/eyes_selenium/selenium/test_dom_capture.py:158]: Move test data to https://github.com/applitools/demo

**Review by skhalymon — APPROVED** [review]

**Pull request merged**